src/lib.rs

```diff
@@ -234,6 +220,20 @@ where
             .map_err(Error::I2C)
             .and(Ok(data[0]))
     }
+
+    fn get_raw_data(&mut self) -> Result<RawData, Error<E>> {
+        // Read raw illuminance data
+        // Use a single transaction to ensure that the data is from the same measurement
+        // (see pg. 17 of datasheet)
```

Comment:
I think it's page 19, right? :smile:

Reply:
![image](https://github.com/Ardelean-Calin/ltr303-rs/assets/33688001/0634fbcd-b3ae-410b-97cf-a118a28f0d6d)
This is the datasheet I'm reading 😅